template/files/default/example.tmpl

```diff
@@ -1,3 +1,6 @@
-# Managed by saltstack
+########################################################################
+# File managed by Salt at <{{ source }}>.
```

Comment:
This will show the `source` of the file in the header of the managed file.

Reply:
This header is used in other formulas such as the [`nfs-formula`](https://github.com/saltstack-formulas/nfs-formula/blob/095db7256fd0ba8ebd5ff5e6ddf01ceeb15560ad/nfs/files/exports).

It's useful enough in normal cases, just to know where the source template resides.  In the TOFS system, I believe this is indispensable, since there are so many potential templates a file could have used.

Reply:
nice